input/vlct/run_HD_linear_wave_test.py

```diff
@@ -0,0 +1,138 @@
+#!/bin/python
+
+# runs VLCT HD Linear Wave Tests
+# - This should probably be transitioned to testing framework once it's created
+# - This script expects to be called from the root level of the repository
+#   OR at the same level where its defined
+#
+# These tests draw loose inspiration from Linear Wave tests used by Athena++.
+# Specifically, this script:
+#   1.) Checks if that L1-norm of the error is appropriate for the inclined
+#       sound wave and 3 variants of the entropy wave for N=16 and N=32
+#       [Unlike Athena++, we are compare L1-norms to precise expected values - 
+#        therefore it probably isn't necessary to use 2 different resolutions]
```

Comment:
Why not? Aren't the two tests complementary, i.e., one should test that a) the errors is getting smaller with higher resolution at the expected rate of convergence and b) that the numerical solution converges to the right analytical one?

Reply:
You're probably right. I have manually done some verification of this. In the future, when we overhaul the testing framework, we should address this.

Reply:
To followup, I didn't originally understand the full context of your comment. I will delete these 2 lines from the file.